runtime/registry.go

```diff
@@ -16,7 +17,7 @@ func (r *Runtime) FindInstance(ctx context.Context, instanceID string) (*drivers
 	return r.Registry().FindInstance(ctx, instanceID)
 }
 
-func (r *Runtime) CreateInstance(ctx context.Context, inst *drivers.Instance) error {
+func (r *Runtime) CreateInstance(ctx context.Context, inst *drivers.Instance, envString string) error {
```

Comment:
Instead of having `envString` as a separate arg, the caller should parse it and set `inst.Env` before calling `CreateInstance`